src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -230,7 +234,9 @@ uct_rc_verbs_iface_query(uct_iface_h tl_iface, uct_iface_attr_t *iface_attr)
 
     iface_attr->cap.flags |= UCT_IFACE_FLAG_EP_CHECK;
     iface_attr->latency.m += 1e-9;  /* 1 ns per each extra QP */
-    iface_attr->overhead   = 75e-9; /* Software overhead */
+
+    /* Software overhead */
+    iface_attr->overhead = UCT_RC_VERBS_IFACE_OVERHEAD;
```

Comment:
we need to take  it from config value (in case it is modified for some platform)

Reply:
It will brake wire compatibility, won't it?

Reply:
we have to decide what to do with wire compat, but we need to take this value from the config, otherwise the config per platform feature does not work

Reply:
Why doesn't it work?
We fill `uct_perf_attr_t` using `uct_iface_estimate_perf`.
Previously we set `perf_attr->send_pre_overhead` to `iface_attr.overhead`.
After this changes we will set this value according the value in the configuration. But I left `iface_attr.overhead` unchanged. Why do you think this is not correct?